### PR TITLE
Layout_changing

### PR DESCRIPTION
**Pull request by annawangkkk**
Did some changes the layout.

### CODE REVIEW

File: AssetMappr/presentation/landingPage.py [hunk 1]
```diff
@@ -11,19 +11,20 @@
 from dash import html
 import dash_bootstrap_components as dbc
 
+
 def makeLandingPage():
```
Comment (mihirbhaskar) [inline AssetMappr/presentation/landingPage.py:15]: Based on how it looks in the pull request review app, the content is not exactly centered, so the position looks a bit funny. Maybe we can also move it a  bit higher so there isn't so much white space on the top? These are very minor points, will wait to see what the rest of the team says as well

File: AssetMappr/presentation/landingPage.py [hunk 2]
```diff
@@ -11,19 +11,20 @@
...
-    
-    return html.Div([
-            html.H1('Welcome to AssetMappr!', style={'color': '#2C3E50'}),
-            html.Br(),
-            html.H4('''We are a group of students that believe in the power of community participation. 
+
+    return html.Div(className='landing', children=[
```
Comment (mihirbhaskar) [inline AssetMappr/presentation/landingPage.py:17]: Maybe just add a quick comment here saying that className refers to the class defined in the .css file so it's clear

File: AssetMappr/presentation/landingPage.py [hunk 3]
```diff
@@ -11,19 +11,20 @@
...
-        
-    
+                    helping planners make better investments for the future.''', id="WelWords"),
+        html.Br(),
+
+        # Link to take users to the main home page of the app
+        dbc.Button('Enter the APP',
+                   href='/home', style={'font-size': '25px'}, id="enterButton")
+
+    ], style={"margin-left": "15px", 'margin-top': '15px', 'margin-right': '40px'})
```
Comment (mihirbhaskar) [inline AssetMappr/presentation/landingPage.py]: If we are setting the margins in the .css class, do we still need this code here?

File: AssetMappr/presentation/layout.py [hunk 4]
```diff
@@ -35,22 +35,25 @@
...
-        
+
         # Layout is split into two overall tabs
         dcc.Tabs([
 
             # Tab 1: Page to view, rate, and upload assets
-            dcc.Tab(label='Tell us about your community', children=[
-                
+            dcc.Tab(id='tab1', label='Tell us about your community', children=[
+
```
Comment (mihirbhaskar) [inline AssetMappr/presentation/layout.py:49]: Haha sorry but maybe the tab text has become too big now! So the layout is looking weird. Maybe we can just change it back to the original size for now, and see what Rick says?

File: AssetMappr/presentation/layout.py [hunk 5]
```diff
@@ -35,22 +35,25 @@
...
                 # This tab uses a grid structure of rows and columns
-                
+
                 # Row 1
                 dbc.Row([
                     dbc.Col(
-                        html.H4('Use the map below to view and select assets'),
```
Comment (mihirbhaskar) [inline AssetMappr/presentation/layout.py:53]: Could we align this text left over the map? Right now the alignment is a bit off

File: AssetMappr/presentation/submitRating.py [hunk 6]
```diff
@@ -17,24 +17,45 @@
...
-                html.Div(id='submit-rating-confirmation')
-            ])
\ No newline at end of file
+        html.Label(['How do you feel about this asset?'],
+                   style={'fontSize': 25}),
+        html.Br(),
+        # Slider bar to score the asset
+        html.H5(
+            'Use the slider below to indicate a rating (0-5):', style={'color': 'dimgray'}),
+        # dcc.Slider(value=2.5, min=0, max=5,
```
Comment (mihirbhaskar) [inline AssetMappr/presentation/submitRating.py]: You can remove code we aren't using anymore - having commented out bits of code makes it tougher to read. We anyways have the old code from the previous commits

File: assets/style.css [hunk 7]
```diff
@@ -0,0 +1,130 @@
+/*
+File: style.css
```
Comment (mihirbhaskar) [inline assets/style.css:2]: For this file, maybe add a comment above every class just to tell the reader where this class is used? E.g. above .landing, '# used for the entire landing page', and above .s, where it is used/what it is used for. Just a simple small comment would do

**Comment by mihirbhaskar** [comment]
Hi Anna - left some specific comments in the code above. Looks good overall! 

I think maybe now seeing it render on the browser, it would be nice if we could reduce font sizes a bit to make sure the features fit neatly on the page without having to scroll - right now the alignment feels a bit off and not as neat as the picture you shared on the Whatsapp

File: AssetMappr/presentation/layout.py (hunk 4 above)
Comment (annawangkkk) [reply AssetMappr/presentation/layout.py:49]: Yeahhhh. tricky thing is that I used the monitor to change our web layout in these days, and now I find everything is soo big when I use my laptop to see our webpage. I am adjusting them now!

**Comment by annawangkkk** [comment]
> Hi Anna - left some specific comments in the code above. Looks good overall!
> 
> I think maybe now seeing it render on the browser, it would be nice if we could reduce font sizes a bit to make sure the features fit neatly on the page without having to scroll - right now the alignment feels a bit off and not as neat as the picture you shared on the Whatsapp

Thanks for your comments! I have committed some new changes.

**Comment by mihirbhaskar** [comment]
Looks great, Anna! Pulling this into the main.

**Pull request merged**